buildconfig/dev-packages/deb/mantid-developer/ns-control

```diff
@@ -38,34 +40,15 @@ Depends: git,
   libqt5x11extras5-dev,
   libqt5opengl5-dev,
   libqt5scintilla2-dev,
-  libpython-dev,
-  python-setuptools,
-  python-numpy,
-  python-scipy,
-  python-qt4-dev,
-  python-qt4-dbg,
-  pyqt5-dev,
-  pyqt5-dev-tools,
-  python-dateutil,
-  python-h5py,
-  python-matplotlib,
-  python-mock,
-  python-psutil,
-  python-qtpy,
-  python-requests,
-  python-sphinx,
-  python-sphinx-bootstrap-theme,
-  python-yaml,
-  ipython-qtconsole (>=1.2.0),
-  texlive,
-  texlive-latex-extra,
-  dvipng,
-  graphviz,
-  doxygen,
+  libpython3-dev,
+  ninja-build,
   python3-setuptools,
   python3-sip-dev,
   python3-pyqt4,
+  python-qt4-dev,
```

Comment:
This seems python 2?

Reply:
This is actually a confusion in naming. You're right in that most packages named `python-` are Python 2 but `python-qt4-dev` is actually Python version agnostic and just contains `.sip` files that are compatible with both versions.

Reply:
I see. Is there a way to put a comment in this file?

Reply:
We can have comments in the file just not within the dependency list. Is a comment that's not near the line much use do you think?

Reply:
No I don't think it will be useful then.